css.editor/licenseinfo.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/css/resources/icons/chrome20-disabled.png</file>
+        <file>src/org/netbeans/modules/css/resources/icons/chrome20.png</file>
+        <file>src/org/netbeans/modules/css/resources/icons/firefox20-disabled.png</file>
+        <file>src/org/netbeans/modules/css/resources/icons/firefox20.png</file>
+        <file>src/org/netbeans/modules/css/resources/icons/ie20-disabled.png</file>
+        <file>src/org/netbeans/modules/css/resources/icons/ie20.png</file>
+        <file>src/org/netbeans/modules/css/resources/icons/opera20-disabled.png</file>
+        <file>src/org/netbeans/modules/css/resources/icons/opera20.png</file>
+        <file>src/org/netbeans/modules/css/resources/icons/safari20-disabled.png</file>
+        <file>src/org/netbeans/modules/css/resources/icons/safari20.png</file>
+        <license ref="CDDL-1.0" />
+        <comment>
```

Comment:
Not sure if this is correct, but someone else will need to comment I guess (e.g. I don't know how where these icons were from in neither repository).

Reply:
My reasoning: the icons are present in the distributed packages from oracle. There are various license statements, but I did not find statements addressing the icons specifically. So I deduce, that they are distributed under the global GPLv2-CDDL license and from that only the CDDL can even be considered for Apache NetBeans.

Reply:
I understand what you are saying, but I don't know if that's correct (sorry).